app/js/components/service_status_interactions.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Why mot use typescript?

Reply:
It has not yet been made available in the project. When it is, this little routine might be a good candidate for a first proof of concept.